examples/boot/bootloader/rp/memory.x

```diff
@@ -0,0 +1,19 @@
+MEMORY
+{
+  /* NOTE 1 K = 1 KiBi = 1024 bytes */
+  BOOT2                             : ORIGIN = 0x10000000, LENGTH = 0x100
+  FLASH                             : ORIGIN = 0x10000100, LENGTH = 24K
+  BOOTLOADER_STATE                  : ORIGIN = 0x10007000, LENGTH = 4K
```

Comment:
why `0x10007000` with a bootloader of `24K`? shouldn't `0x10006000` suffice, if you do
`FLASH                             : ORIGIN = 0x10000100, LENGTH = 24K - 0x100`

Reply:
Agreed!